assets/terraform-modules/aws/flatcar-linux/kubernetes/security.tf

```diff
@@ -168,6 +168,19 @@ resource "aws_security_group_rule" "controller-egress" {
   ipv6_cidr_blocks = ["::/0"]
 }
 
+resource "aws_security_group_rule" "controller-node-local-dns-metrics-port" {
+  count = var.enable_node_local_dns ? 1 : 0
+
+  security_group_id = aws_security_group.controller.id
+
+  type                     = "ingress"
+  protocol                 = "tcp"
+  from_port                = 9253
```

Comment:
Why do we need to expose it if each node should only access this locally? This traffic shouldn't be routed through the external interface, no?

Reply:
Yep port 53 is meant to be used locally. But port 9253 is a metrics port and prometheus needs access to it. Now prometheus could be running anywhere so we have to expose this metrics port on all nodes.

Reply:
`metrics-port`, got it :+1: